idaes/models_extra/gas_solid_contactors/unit_models/fixed_bed_0D.py

```diff
@@ -26,6 +30,7 @@
     UnitModelBlockData,
 )
 from idaes.core.util.config import is_physical_parameter_block
+from idaes.core.util.exceptions import InitializationError
```

Comment:
I do not see this being used later in the new code.

Reply:
Ah, thanks - I'll clean up the imports. I had attempted to implement an `InitializationError` check similar to the `PressureChanger` initialization on [lines 759-777](https://github.com/IDAES/idaes-pse/blob/a7680abb68c179e7c3abd27ddcbfe5a3f39747c3/idaes/models/unit_models/pressure_changer.py#L759), but found that inserting an extra solve statement in order to check the termination condition spawned other issues. Please let me know if you think the FB0D should use the `InitializationError` or if it is simple enough not to need it.

Reply:
You should have the `InitialziationError`. However, do you need an additional solve to do it? An initialization routine should always finish with ha solve of the full model (unless it happens to have no active constraints), so there should generally be a solution for you to check. The only case where you can skip the check is if you have no active constraints (and even then it should be conditional on there being no active constraints).

Reply:
Currently, the initialization routine does not have any full model solves - it initializes the gas/solid thermophysical properties and freezes them, initializes the reactions properties and releases the property states. I would need to add a Step 2 which is a full solve similar to the linked PressureChanger script.